frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelineFilterBar.ts

```diff
@@ -32,15 +32,15 @@ class PipelineRunFilterBar extends PipelineFilterBar {
   }
 
   private findActiveRunsTable() {
-    return cy.findByTestId('active-runs-table');
+    return cy.findByTestId('runs-runs-table');
```

Comment:
I think the concept of "active" can still be associated with the first tab since "runs" is technically an umbrella that "archived" runs can exist in.

I get that the tab title changed to "Runs" and that's what led down this path, but the title of the tab doesn't necessarily have to dictate the naming conventions internally here.

I'm curious about others' opinions on this. There are a lot of code changes here to naming that seems like could have been avoided.

Reply:
Hmm, this is a good suggestion. I can see if I can update this back.